src/rgw/rgw_rados.cc

```diff
@@ -3322,7 +3322,7 @@ int RGWRados::get_required_alignment(const rgw_pool& pool, uint64_t *alignment)
     return 0;
   }
 
-  uint64_t align;
+  uint64_t align = 0;
```

Comment:
@agutikov if it's not initialized by the next call then the fix should be in librados